content/blog/haven-2-4-0/index.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+output: hugodown::hugo_document
+
+slug: haven-2-4-0
+title: haven 2.4.0
+date: 2021-04-15
+author: Hadley Wickham
+description: >
+    This version provides much improved `labelled_spss()` support, improved
+    date-time handling, latest ReadStat, and a bunch of other small 
+    improvements.
+
+photo:
+  url: https://unsplash.com/photos/SHA85I0G8K4
+  author: Evgeni Tcherkasski
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [haven]
+---
+
+We're delighted to announce the release of [haven](https://haven.tidyverse.org) 2.4.0. haven allows you to read and write SAS, SPSS, and Stata data formats from R, thanks to the wonderful [ReadStat](https://github.com/WizardMac/ReadStat) C library written by [Evan Miller](https://www.evanmiller.org/).
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("haven")
+```
+
+This blog post will show off the most important changes to the package; you can see a full list of changes in the [release notes](https://github.com/tidyverse/haven/releases/tag/v2.4.0)
+
+```{r setup}
+library(haven)
+```
+
+## `labelled_spss()` and `labelled()`
+
+`labelled_spss()` gains full vctrs support thanks to the hard work of [Danny Smith](https://github.com/gorcha). This means that `labelled_spss()` objects should now work seamlessly in dplyr 1.0.0, tidyr 1.0.0.
```

Comment:
```suggestion
`labelled_spss()` gains full vctrs support thanks to the hard work of [Danny Smith](https://github.com/gorcha). This means that `labelled_spss()` objects should now work seamlessly with dplyr 1.0.0, tidyr 1.0.0.
```

Reply:
I _think_ this is how we typically construct this (x works _with_ dplyr…), but both are grammatically sound.